.rubocop_todo.yml

```diff
@@ -26,22 +26,6 @@ Lint/UnreachableCode:
   Exclude:
     - 'spec/raven/raven_spec.rb'
 
-# Offense count: 4
-# Cop supports --auto-correct.
-# Configuration parameters: IgnoreEmptyBlocks.
-Lint/UnusedBlockArgument:
-  Exclude:
-    - 'spec/raven/integrations/rack_spec.rb'
-    - 'spec/raven/raven_spec.rb'
-
-# Offense count: 2
-# Cop supports --auto-correct.
-# Configuration parameters: AllowUnusedKeywordArguments, IgnoreEmptyMethods.
-Lint/UnusedMethodArgument:
-  Exclude:
-    - 'lib/raven/backports/uri.rb'
```

Comment:
I'd prefer not to touch anything in the backports folder.

Please add `'lib/raven/backports/uri.rb'` to `rubocop.yml` as a globally ignored file, and then drop the changes to that file from this PR.